src/models/StyledNativeComponent.js

```diff
@@ -24,6 +24,7 @@ export default (constructWithOptions: Function) => {
       theme: null,
       generatedStyles: undefined,
     }
+    root: Object
```

Comment:
Should probably be grouped with the static flow properties (types) above, instead of the class instances (values)